lib/tether/toolbox.go

```diff
@@ -127,8 +127,8 @@ func (t *Toolbox) kill(name string) error {
 		return fmt.Errorf("failed to kill container: process not found")
 	}
 
-	session.m.Lock()
-	defer session.m.Unlock()
+	session.Lock()
```

Comment:
every use of session.Cmd.Process in toolbox now needs a nil check - the extension is now being initialized before process launch.